aws/alarms/cloudwatch.tf

```diff
@@ -453,6 +453,38 @@ resource "aws_cloudwatch_metric_alarm" "generate_temporary_token_api_failure" {
   alarm_actions     = [var.sns_topic_alert_warning_arn]
   ok_actions        = [var.sns_topic_alert_ok_arn]
 
+  tags = {
+    (var.billing_tag_key) = var.billing_tag_value
+    Terraform             = true
+  }
+}
+
+resource "aws_cloudwatch_log_metric_filter" "request_temporary_token_api_using_unauthorized_email_address" {
+  name           = "RequestTemporaryTokenApiUsingUnauthorizedEmailAddress"
+  pattern        = "An email address with no access to any form has been locked out"
```

Comment:
CloudWatch patterns are a little tricky... when you want to match an exact phrase you need to also include escaped double quotes.
`pattern = "\"An email address with no access to any form has been locked out\""`

Reply:
I did copy what already existed in the codebase (other alarms). In fact this is only a part (static part) of the full log that will be published from the client. I will change it and we can always modify the pattern in the future if needed :)

Reply:
I know, I quickly saw that it was also used elsewhere.  The pattern will work but instead of the exact phrase it looks for "An" + "email" + "address" +  ... etc.